crates/next-custom-transforms/src/transforms/server_actions.rs

```diff
@@ -932,11 +932,13 @@ impl<C: Comments> VisitMut for ServerActions<C> {
     }
 
     fn visit_mut_fn_expr(&mut self, f: &mut FnExpr) {
```

Comment:
is `fn_expr` the old thing we need to handle here? i think technically this would also work, but idk if we're covering it somehow
```
async function test() {
  'use server'
  const MyClass = class {
    x = 1;
    foo() { return this.x }
  }
  return (new MyClass()).foo()
}
```